.eslintrc

```diff
@@ -15,6 +15,7 @@
     "consistent-return": 0,
     "no-param-reassign": 0,
     "comma-dangle": 0,
+    "global-require": "off",
```

Comment:
How does this fix the issue with flagging require statements?

According to the `eslint` [documentation](http://eslint.org/docs/rules/global-require), this rule is to `Enforce require() on the top-level module scope (global-require)`.

PS: Let's try to use the number representations for enforcing rules (`0`, `1`, `2`).